src/UserStory/Home.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react'
 import { Link } from 'react-router-dom'
-import bee from '../bee.png'
+import bee from '../Images/bee.png'
```

Comment:
It's better if you use css to display your image so that we don't care about the image source when it's changed

Reply:
More recommend: you should try to apply SASS as a css pre-compiler

Reply:
Used css instead of the previous method, i did some reasearch on SASS, i will try and apply it from now on.